config/scripts/terminal-output-watt-ab-macos.sh

```
@@ -0,0 +1,153 @@
+  done
+  sleep 4
+  local RENDERER_PID GPU_PID
+  RENDERER_PID=$(find_pid renderer "$dir")
+  GPU_PID=$(find_pid gpu-process "$dir")
+  if [ -z "$RENDERER_PID" ]; then
+    echo "phase '$name': could not find renderer PID"
+    kill "$SPEC_PID" 2>/dev/null || true
+    return 1
+  fi
+  echo "-- sampling renderer pid=$RENDERER_PID gpu pid=${GPU_PID:-none} for ${SAMPLES}s --"
+  local top_args=(-l $((SAMPLES + 1)) -s 1 -stats pid,cpu,idlew,power -pid "$RENDERER_PID")
```

Shellcheck (0.11.0)

[warning] Use spaces, not commas, to separate array elements.

(SC2054)